src/web/core/src/test/java/org/geoserver/web/data/resource/ResourceConfigurationPageTest.java

```diff
@@ -634,9 +622,11 @@ public void testWMTSOtherCRSUrnFormat() throws IOException {
                         "publishedinfo:tabs:panel:theList:0:content:referencingForm:nativeSRS:popup:content:table:listContainer:items:3:itemProperties:0:component:link:label");
 
         // checks that they have been properly displayed with not urn format being cut
-        assertEquals("3857", epsgComponent1.getDefaultModel().getObject());
-        assertEquals("urn:ogc:def:crs:EPSG::3857", epsgComponent2.getDefaultModel().getObject());
-        assertEquals("urn:ogc:def:crs:EPSG::900913", epsgComponent3.getDefaultModel().getObject());
+        assertEquals("urn:ogc:def:crs:EPSG::3006", epsgComponent1.getDefaultModel().getObject());
```

Comment:
Confused here too. Are these checks more in line with what's in the capabilities document? (and was the old caps document fully replaced, and why?)

Reply:
GT was previously using this code to extract srs name https://github.com/geotools/geotools/blob/cffe95a3a62bd12cb341b10cc498f1b81ebfa7af/modules/extension/wms/src/main/java/org/geotools/ows/wms/Layer.java#L798
of a layer. This in turn caused the ui to show up among the native srs, thus the server supported ones, also srs definition actually not present in the wmts getCapabilities document.

In the previous test we had a capaibilites document with an EPSG:3857 supported srs only, and the code was showing also as natively supported also urn:ogc:def:crs:EPSG::3857 and urn:ogc:def:crs:EPSG::900913. Once the fix happens on geotools, the test for the proper display of urn:ogc:def:crs:EPSG:: definition could not use these same capabilities since contains just an EPSG definition.

The new capabilities document instead is declaring two srs one of which is using the urn:ogc:def:crs:EPSG:: notation, so that it was possibile to keep the already present tests to check the proper display of it.